README.md

```diff
@@ -364,5 +364,22 @@ See the `config/stack/overrides` directory for details about where the
 overrides live. See the `stack.Makefile` for details about how
 `local-build` works.
 
+### Setup RBAC
+
+We can setup extra permissions to grant access to the resources which are not part of our stack. This can be done by specifing the permissions in the CustomResourceDefinition YAML of our custom resource.
```

Comment:
Nitpick/typo: `specifing` should be `specifying`

Reply:
That's a typo alright. Thanks for pointing it out :D

Reply:
Fixed